cards.html

```diff
@@ -4,9 +4,121 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css"/>
+    <script src="https://kit.fontawesome.com/0a707a45d9.js" crossorigin="anonymous"></script>
+
+    <title>Cards</title>
 </head>
 <body>
-    
+    <!-- CARD CONTAINER-1 -->
+    <section class="card-container">
+        <div class="content">
+        <img class="image" src="assets/shirt.jpg" alt="img"/>
+        <div class="heading">
```

Comment:
You can be specific to the class name. Such as `card-heading`.
This will not raise conflict issues in a future projects.

Reply:
Done